Source/RestKit/JSON.swift

```diff
@@ -98,18 +105,42 @@ public enum JSON: Equatable, Codable {
         }
     }
 
-    /// Initialize a JSON value from an encodable type.
-    public init<T: Encodable>(from value: T) throws {
+    public static func encoder() -> JSONEncoder {
```

Comment:
This is fine, but I wonder whether it might be more flexible if it was a computed property instead. Something like this:

```swift
public static var encoder: (() -> JSONEncoder) = {
    let encoder = JSONEncoder()
    ...
    return encoder
}
```

That way the user can change the configuration of the encoder if they want?

Reply:
Good suggestion.  That will also avoid constantly constructing and destructing these encoders and decoders. I'll make that change.